datahub-web-react/src/app/permissions/AvatarsGroup.tsx

```diff
@@ -47,6 +49,22 @@ export default function AvatarsGroup({ users, groups, entityRegistry, maxCount =
                         />
                     </div>
                 ))}
+            {roles &&
+                roles.length > 0 &&
+                (roles || [])?.map((role, key) => (
```

Comment:
you already check whether roles.length > 0 so you don't need to do this: `(roles || [])?`

Please simplify

Reply:
agreed